### DISCUSSION TITLE
pikpak网盘无法正常使用

### DISCUSSION THREAD

**Discussion by ninmoe** [AlistGo/alist#508] [Q&A]
下载文件，在线播放视频都不行，你的演示站里的也是不行的，链接：https://alist.xhofe.top/PikPak/PikPak Tutorial.mp4
移动网络

**Comment by xhofe** [comment]
你有没有想过是自己网络问题

**Reply by ninmoe** [reply]
我找了好几个朋友的移动卡/wifi都不行啊，是移动网络的问题吗？联通和电信没问题啊

**Reply by xhofe** [reply]
不来就不是国内的 墙了不是很正常？
挂个代理试试就知道了
